### DISCUSSION TITLE
Show the service stack UUID clearly in the UI

### DISCUSSION THREAD

**Discussion by nuxwin** [coollabsio/coolify#3584] [Feature Requests]
Good morning,

We added a ressource using Docker compose, resulting to a service UUID mcw4c8w4o08cswg4ks0okwkk

Could it be possible to clearly show the service UUID in the UI (a new readonly field), so we can retrieve it easily ?

We added some custom traefik labels in our compose definition and we had to search for the service UUID. Of course, it is in the URL, and we can also grab it from the deployable compose but, it could be more user friendly to see it in the main configuration tab.

I know that when you create the service, it is added in the name field but, as you can see in the screen below, the service name field is editable and so, the UUID can be removed.

Basically put, a new readonly Service UUID field, placed before the Service name field (top of it), would be welcome.

Thank you.

![Capture d’écran 2024-09-26 à 13 09 37](https://github.com/user-attachments/assets/484c90a8-ad35-4156-9bdf-e024f17339ef)

**Comment by qverkk** [comment]
This would be awesome. Also it'd be nice if we could name our services / applications / databases. 

When creating a new resource you'd be prompted for the service name, by default it'd be an uuid. But the user would have the option to set a custom name for the resource/container, which later on would be easier to configure other services.

Example:
- The user creates a postgres database for his spring backend service, he names the postgres instance: postgres-spring-database
- The user configures the spring-application and sets the DB_HOST to jdbc:postgresql://postgres-spring-database:5432/postgres
-- Instead of the usual config like jdbc:postgresql://tcs044g4w4g88sg0c0c0o4w0:5432/postgres

This would be more readable an concise imo. The same would apply for redis:
- env var REDIS_HOST=spring-backend-redis
- instead of REDIS_HOST=tsg04oc4cowg404c80c4sggg
